src/entity/posts/Post.ts

```diff
@@ -230,4 +230,8 @@ export class Post {
     lazy: true,
   })
   public relatedPosts: Promise<PostRelation[]>;
+
+  @Column({ type: 'text', nullable: false, unique: true })
```

Comment:
I think there is also syntax:

```js
@Column({
    generatedType: 'STORED',
    asExpression: 'sql expression here'
})
```

Reply:
Oh cool!
Didnt find that as quickly 🫤

Reply:
@capJavert Modified :) 
Also added the metadata sql part